src/Video/RenderSurface.cpp

```diff
@@ -84,14 +57,18 @@ ReadWriteTexture* RenderSurface::GetColorTexture() const {
     return colorTexture[which];
 }
 
-ReadWriteTexture* RenderSurface::GetExtraColorTexture() const {
-    return extraColorTexture[which];
+void RenderSurface::Swap() {
+    which = 1 - which;
 }
 
-ReadWriteTexture* RenderSurface::GetExtraDepthTexture() const {
-    return extraDepthTexture[which];
-}
+void RenderSurface::Clear() const {
+    depthFrameBuffer->BindWrite();
+    depthFrameBuffer->Clear();
+    depthFrameBuffer->Unbind();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetNormalTexture' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)